lib/extension/homeassistant.ts

```diff
@@ -1020,6 +1020,16 @@ export default class HomeAssistant extends Extension {
                     Object.assign(extraAttrs, {device_class: 'energy', state_class: 'total_increasing'});
                 }
 
+                // If a variable includes A or mA, mark it as energy
+                if (firstExpose.unit && ['A', 'mA'].includes(firstExpose.unit)) {
+                    Object.assign(extraAttrs, {device_class: 'current', state_class: 'measurement'});
+                }
+
+                // If a variable includes mW, W, kW, MW, GW, TW, mark it as energy
+                if (firstExpose.unit && ['mW', 'W', 'kW', 'MW', 'GW', 'TW'].includes(firstExpose.unit)) {
```

Comment:
```suggestion
                if (firstExpose.unit && ['mW', 'W', 'kW'].includes(firstExpose.unit)) {
```

It's unlikely that these will ever be there. Also could you make it and `if/else if` statement?